application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -641,6 +641,17 @@ define('diagram-link-editor', [
           // Keep only the text content.
           str = $('&lt;div/&gt;').html(str).text() || this.foAltText;
         }
+        // Update links inside foreignObject to use absolute URLs, since the generated SVG needs to be portable.
+        var linkNodes = $(fo).find('a');
+        linkNodes.filter(function(index) {
```

Comment:
Why use ``filter`` and not ``each``? ``filter`` makes sense if you do something with the result. Also, instead of using the index to access the item you can use ``$(this)``.